moped-editor/src/views/projects/projectView/DateFieldEditComponent.js

```diff
@@ -5,12 +5,15 @@ import { parseISO, format } from "date-fns";
 /**
  * DateFieldEditComponent - renders a Date type Calendar select
  * @param {object} props - Values passed through Material Table `editComponent`
+ * @param {function} props.onChange - callback function to update the value
+ * @param {string} props.value - the current value
+ * @param {object} props.textFieldProps - additional props to pass to the picker's TextField
  * @return {JSX.Element}
  * @constructor
  */
 
 const DateFieldEditComponent = React.forwardRef(
-  ({ onChange, value, ...props }, ref) => {
+  ({ onChange, value, textFieldProps, ...props }, ref) => {
```

Comment:
This new prop is added so we can pass error state and error message to the TextField within the MUI MobileDatePicker. The validation error state and messages provided by React Hook Form/Yup are passed in the component form.